Lib/test/test_termios.py

```diff
@@ -12,12 +12,10 @@ class TestFunctions(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        try:
-            cls.stream = open('/dev/tty', 'wb', buffering=0)
-        except OSError:
-            raise unittest.SkipTest("Cannot open '/dev/tty'")
+        cls.master_fd, cls.fd = os.openpty()
```

Comment:
Some systems might not have it: `.. availability:: Unix, not Emscripten, not WASI.`

Other tests use something like `@unittest.skipUnless(hasattr(os, 'openpty'), 'need os.openpty()')`

Reply:
Are there platforms that have termios, but not os.open? I'll add the guard, but first I want to run tests on buildbots to see what other issues can be here.

Reply:
There is one such platform: WebAssembly.